openapi/index.yaml

```diff
@@ -1,6 +1,12 @@
 swagger: "2.0"
 info:
+  title: "IO APIs, shared specs"
   version: "0.1"
 paths:
+  "/dummy":
+    get:
+      responses:
+        "200":
+          description: "ok"
 definitions:
-  $ref: definitions.yaml
\ No newline at end of file
+  $ref: definitions.yaml
```

Comment:
This is not OAS/Swagger valid. `$ref` can only be used to reference a single schema and not a full remote file.

I'm checking the `oval` package you're using.